### PR TITLE
SECURITY.md: Add Security Monitoring

### PR DESCRIPTION
**Pull request by claudijd**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -175,4 +176,20 @@ The caveats of such an approach include the following:
...
 - Speed to shutdown is limited by speed at which goverance can be passed to accept the temporary bug fix (slower for unknown cases and faster for known cases)
-- Restoring after a shutdown will require a secondary governance action to either repoint the proxy contract to a non-shutdown implementation (known cases) or to revert the temporary patch and apply the long term patch (unknown cases)
\ No newline at end of file
+- Restoring after a shutdown will require a secondary governance action to either repoint the proxy contract to a non-shutdown implementation (known cases) or to revert the temporary patch and apply the long term patch (unknown cases)
+
+## Security Monitoring
+
+The Wormhole project expects all Guardians develop and maintain their own security monitoring strategies.  This expectation is based on the value of having heterogeneous monitoring strategies across the Guardians set as a function of Wormhole's defense in depth approach, increasing the likelihood of detecting fraudulent activity.
+
+Wormhole Guardians should aim to capture all of the following domians with their monitoring strategies:
```
Comment (tbjump) [inline SECURITY.md:185]: domains

File: SECURITY.md [hunk 2]
```diff
@@ -175,4 +176,20 @@ The caveats of such an approach include the following:
 
 - Speed to shutdown is limited by speed to develop the temporary bug fix (only for the unknown cases, known cases won't require development)
 - Speed to shutdown is limited by speed at which goverance can be passed to accept the temporary bug fix (slower for unknown cases and faster for known cases)
-- Restoring after a shutdown will require a secondary governance action to either repoint the proxy contract to a non-shutdown implementation (known cases) or to revert the temporary patch and apply the long term patch (unknown cases)
\ No newline at end of file
+- Restoring after a shutdown will require a secondary governance action to either repoint the proxy contract to a non-shutdown implementation (known cases) or to revert the temporary patch and apply the long term patch (unknown cases)
+
+## Security Monitoring
+
+The Wormhole project expects all Guardians develop and maintain their own security monitoring strategies.  This expectation is based on the value of having heterogeneous monitoring strategies across the Guardians set as a function of Wormhole's defense in depth approach, increasing the likelihood of detecting fraudulent activity.
```
Comment (tbjump) [inline SECURITY.md:183]: "across the Guardians set" --> "across the Guardian set"

**Review by tbjump — DISMISSED** [review]
some nits but lgtm

**Review by njkumr — DISMISSED** [review]

**Review by tbjump — APPROVED** [review]
lgtm

**Pull request merged**